mayastor/Cargo.toml

```diff
@@ -23,6 +24,10 @@ path = "src/bin/uring-support.rs"
 name = "mayastor-client"
 path = "src/bin/cli/cli.rs"
 
+[[bin]]
+name = "dumb"
```

Comment:
let us give it a better name @jkryl -- how about rpc? or jrpc (j for json?)

Reply:
Tiago requested a better name too :-) So we decided to go forward with "jsonrpc" name.